core/services/offchainreporting2/delegate.go

```diff
@@ -111,6 +105,13 @@ func (d Delegate) ServicesForSpec(jobSpec job.Job) (services []job.Service, err
 		"DatabaseTimeout", lc.DatabaseTimeout,
 	)
 
+	// We buffer so that ocr2 providers may signal and continue working
+	// upon detecting contract config.
+	contractReady := make(chan struct{}, 1)
```

Comment:
Instead of sending which requires buffering, would it be simpler to close the chan?

Reply:
yeah good call